pkgs/applications/graphics/gimp/default.nix

```diff
@@ -23,7 +23,7 @@ in stdenv.mkDerivation rec {
     freetype fontconfig lcms libpng libjpeg poppler poppler_data libtiff openexr
     libmng librsvg libwmf zlib libzip ghostscript aalib shared-mime-info libwebp libheif
     python pygtk libexif xorg.libXpm glib-networking libmypaint mypaint-brushes
-  ] ++ stdenv.lib.optionals stdenv.isDarwin [ AppKit Cocoa gtk-mac-integration ]
+  ] ++ stdenv.lib.optionals stdenv.isDarwin [ AppKit Cocoa gtk-mac-integration-gtk2 cf-private ]
```

Comment:
Why is cf-private needed? Could you add a comment for it.

Reply:
Done

Reply:
Thanks, that will help to cleanup these workarounds after the swift-corelibs are fixed.